dispatcher/backend/src/tests/integration/routes/schedules/test_zimit.py

```diff
@@ -0,0 +1,111 @@
+        assert response.status_code == 201
+
+    # below test is green only if ZIMIT_DISABLE_ZIM_FILENAME_CHECK is set
+    # def test_create_zimit_schedule_bad_name_ok(
+    #     self, client, access_token, garbage_collector
+    # ):
+    #     schedule = {
+    #         "name": "zimit_test_bad_name_ok",
+    #         "category": "other",
+    #         "enabled": False,
+    #         "tags": [],
+    #         "language": {
+    #             "code": "fr",
+    #             "name_en": "French",
+    #             "name_native": "Français"
+    #         },
+    #         "config": {
+    #             "task_name": "zimit",
+    #             "warehouse_path": "/other",
+    #             "image": {"name": "openzim/zimit", "tag": "1.0.0"},
+    #             "monitor": False,
+    #             "platform": None,
+    #             "flags": {
+    #                 "name": "acme",
+    #                 "url": "https://www.acme.com",
+    #                 "zim-file": "bad_name",
+    #             },
+    #             "resources": {"cpu": 3, "memory": 1024, "disk": 0},
+    #         },
+    #         "periodicity": "quarterly",
+    #     }
+
+    #     url = "/schedules/"
+    #     response = client.post(
+    #         url, json=schedule, headers={"Authorization": access_token}
+    #     )
+    #     response_data = response.get_json()
+    #     print(response_data)
+    #     if "_id" in response_data:
+    #         garbage_collector.add_schedule_id(response_data["_id"])
+    #     assert response.status_code == 201
+
+    # below test becomes red if ZIMIT_DISABLE_ZIM_FILENAME_CHECK is set
+    def test_create_zimit_schedule_bad_name_nok(
+        self, client, access_token, garbage_collector
+    ):
+        schedule = {
+            "name": "zimit_test_bad_name_nok",
+            "category": "other",
+            "enabled": False,
+            "tags": [],
+            "language": {"code": "fr", "name_en": "French", "name_native": "Français"},
+            "config": {
+                "task_name": "zimit",
+                "warehouse_path": "/other",
+                "image": {"name": "openzim/zimit", "tag": "1.0.0"},
+                "monitor": False,
+                "platform": None,
+                "flags": {
+                    "name": "acme",
+                    "url": "https://www.acme.com",
+                    "zim-file": "bad_name",
+                },
+                "resources": {"cpu": 3, "memory": 1024, "disk": 0},
+            },
+            "periodicity": "quarterly",
+        }
+
+        url = "/schedules/"
+        response = client.post(
+            url, json=schedule, headers={"Authorization": access_token}
+        )
+        response_data = response.get_json()
+        print(response_data)
+        if "_id" in response_data:
+            garbage_collector.add_schedule_id(response_data["_id"])
+        assert response.status_code == 400
```

Comment:
Would it be possible (while remaining clear and easy to maintain) to alter schema in `get_offliner_schema` so that the flag becomes _dynamic_?

Reply:
> Change your test to have different expectations based on the value of the flag instead of keeping a commented copy that will never be used.

Did I get it correctly that you mean that we will still have to modify flag value more or less manually to check both test cases, but at least we won't have commented code anymore? This makes sense to me, but I'm not sure I got you correctly.

Reply:
Never mind, I achieved to make both test run ok, my bad Python knowledge was the cultprit. Same solution will probably apply to the other PR about worker IPs, do not review it for now, I will fix it and keep you informed.

Reply:
OK